frontend/src/services/requestService.js

```diff
@@ -0,0 +1,68 @@
+import axios from 'axios';
+
+axios.defaults.headers.common = {
```

Comment:
don't modify global object, it's better to create axios instance, and define default params there